readme.md

```diff
@@ -128,6 +128,21 @@ class TrafficLight implements ColorSchemeInterface
 
 Please refer to [the `bramus/ansi-php` documentation](https://github.com/bramus/ansi-php) to define your own styles and colors.
 
+## Custom Formatting
+You may with to colorize only a portion of the output.  For example
+
+    [add image here]
```

Comment:
I thought you might want to match the color scheme of the other screen shots, so I didn't include an image here.  But in the test I included the code for generating, commented out.